projects/gloo/pkg/plugins/kubernetes/eds.go

```diff
@@ -311,6 +311,16 @@ func filterEndpoints(
 			errorsToLog = append(errorsToLog, fmt.Sprintf("upstream %v: port %v not found for service %v", usRef.Key(), spec.GetServicePort(), spec.GetServiceName()))
 			continue
 		}
+
+		// Istio uses the service's port for routing requests
+		if istioIntegrationEnabled {
+			hostname := fmt.Sprintf("%v.%v", spec.GetServiceName(), spec.GetServiceNamespace())
+			key := Epkey{hostname, uint32(kubeServicePort.Port), spec.GetServiceName(), spec.GetServiceNamespace(), usRef}
```

Comment:
Nit: For readability, could we call out the field names in this key?
```suggestion
                        key := Epkey{
				Address:     hostname,
				Port:        uint32(kubeServicePort.Port),
				Name:        spec.GetServiceName(),
				Namespace:   spec.GetServiceNamespace(),
				UpstreamRef: usRef,
			}
			copy
```